src/main/java/session/FormSession.java

```diff
@@ -130,7 +140,7 @@ public FormSession(UserSandbox sandbox, FormDef formDef, String username, String
     private void initialize(boolean newInstance, Map<String, String> sessionData) {
         CommCarePlatform platform = new CommCarePlatform(2, 27);
         FormplayerSessionWrapper sessionWrapper = new FormplayerSessionWrapper(platform, this.sandbox, sessionData);
-        FormplayerConfigEngine.setupStorageManager(username, "dbs");
+        FormplayerConfigEngine.setupStorageManager("test", "dbs/" + domain + "/" + username);
```

Comment:
if so it's unintuitive to have test code being run outside of tests, can you leave a comment?